### PR TITLE
merger_config: add missing strategies

### PR DESCRIPTION
**Pull request by michamos**
* This adds two missing strategies, `ManualMergeOperations` for merging
by hand two existing records, and `ArxivOnPublisherOperations` for an
arXiv harvest on top of a publisher record.
* This also required adapting the strategy selection function to ensure
that the manual merger configuration gets called when needed.

Signed-off-by: Micha Moskovic <michamos@gmail.com>

### CODE REVIEW

**Comment by ammirate** [comment]
I'm going to add some tests before merging this

File: inspire_json_merger/inspire_json_merger.py [hunk 1]
```diff
@@ -135,3 +138,13 @@ def get_acquisition_source(json_obj):
         return json_obj['acquisition_source']['source']
     except KeyError:
         return 'arxiv'
```
Comment (ammirate) [inline inspire_json_merger/inspire_json_merger.py]: @michamos is this assumption still valid?

Comment (michamos) [reply inspire_json_merger/inspire_json_merger.py]: I don't know. Do you remember why you wrote this?

Comment (ammirate) [reply inspire_json_merger/inspire_json_merger.py]: as discussed IRL, the right thing to do when `acquisition_source` is missing is assuming that the record comes from a publisher.

**Pull request merged**